Database/index.js

```diff
@@ -1,11 +1,13 @@
 const mysql = require('mysql');
 
-// Create a database connection and export it from this file.
-// You will need to connect with the user "root", no password,
-// and to the database "yelp".
+// // Create a database connection and export it from this file.
+// // You will need to connect with the user "root", no password,
```

Comment:
This direction may be in slight contradiction to the usage with Docker / aws since it (the connection) does use a password.